billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/documents/PTGenericInvoiceIssuingHandler.java

```diff
@@ -110,13 +112,17 @@ protected <D extends AbstractDAOPTGenericInvoice<T>> T issue(final T document, f
         String sourceHash = GenerateHash.generateSourceHash(invoiceDate, systemDate, formattedNumber,
                 document.getAmountWithTax(), previousHash);
 
+		if (invoiceSeriesEntity.getSeriesUniqueCode().isPresent()) {
```

Comment:
Can you review the indentation format used for the proposed changes? It is currently indented with tabs instead of the spaces used on the remaining project